bin/gen-drone.sh

```diff
@@ -26,18 +26,22 @@ fi
 readonly webhook=$(echo "$values" | yq r - "alerts.$receiver.$key")
 
 function template_drone_config() {
-  local targetPath="$ENV_DIR/env/clouds/${CLOUD}/${CLUSTER}/.drone.yml"
-  local image_tag=$(otomi_image_tag)
+  local targetPath="$ENV_DIR/env/.drone.yml"
```

Comment:
this should go to the root of the repo as it is the default and will immediately be recognized then

Reply:
yes! Then we do not make an additional config at drone. Thanks for reminding me about that.